EIPS/eip-3664.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 3664
+title: ERC-3664 next generation game NFT
+author: TylerZ(@tylerztl), TY(@nimocat)
+discussions-to: <URL>
```

Comment:
Please include a discussions URL. This is required before merging as a draft. We generally prefer people to create a thread on https://ethereum-magicians.org.